### ISSUE TITLE
LAR data files for Snapshot and Dynamic data have different specifications

### ISSUE THREAD

**Issue by awolfe76** [cfpb/hmda-platform#2061]
_Created from https://github.com/cfpb/hmda-pub-ui/issues/192_

---

Should we standardize our LAR data schemas for 2017 so that users can more easily switch between the two?

Differences:
Dynamic has record ID and does not have As of Date, Edit Status, Sequence Number, or Application Date Indicator.

Some column names differ slightly between the two file specifications:
Occupancy vs Owner Occupancy
Preapproval vs Preapprovals
Action Type vs Type of Action Taken
Purchaser Type vs Type of Purchaser

**Comment by keileyo** [comment]
I have some concerns about this. The as of date, edit status, sequence number and application date indicator are all blank fields that we added just to standardize snapshot data with the Fed's snapshot file.  At this point, we have been publishing the dynamic data without them for 3+ months without any request from users to add them. 

Changing the schema of the file without notification will also cause issues for any users who have set up automated pulls of the dynamic data (a la January APOR issues). I do not think that the risks here are outweighed by the perceived benefits.

**Comment by nickgrippin** [comment]
I've already talked to @Kibrael and @awolfe76 about this issue.  We're re-opening the [pub-ui issue](https://github.com/cfpb/hmda-pub-ui/issues/192) in order to add a small note to one or both pages explaining the difference in schemas.  

We had also talked about updating the column names to match each other, but that might cause issues with anyone doing automated pulls.  I'd be fine with leaving the column names as-is, since I don't think we've received any negative feedback.
